src/main/kotlin/nl/info/zac/app/zaak/ZaakRestService.kt

```diff
@@ -1100,20 +1099,77 @@ class ZaakRestService @Inject constructor(
 
     private fun datumWaarschuwing(vandaag: LocalDate, dagen: Int): LocalDate = vandaag.plusDays(dagen + 1L)
 
-    private fun assignLoggedInUserToZaak(
+    private fun validateUserInGroup(userName: String?, groupId: String) {
```

Comment:
Sure the userName should be nullable? Why would you even want to call this function without a username? Seems a bit weird to me.

Reply:
Ohh, you're completely right. I extracted this bit from existing code but it does not make sense for a separate function. Fixing ...